<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>977</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenFoodFacts Food Substitutions (OpeClassRooms projet-5)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/assets/images/icon.png</normaloff>:/images/assets/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_user_mode">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="users_box">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>250</height>
         </size>
        </property>
        <property name="title">
         <string>Utilisateur</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="signin">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sign-in</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="signup">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sign-up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">border-image: url(:/images/assets/images/home 1.png);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/assets/images/openfoodfacts+oc.png);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="substitute_mode_box">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>250</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Condensed</family>
          <pointsize>11</pointsize>
          <underline>true</underline>
          <stylestrategy>NoAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="title">
         <string>Reherche de substitution d'aliment</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="openfoodfacts_mode">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>false</underline>
             <stylestrategy>NoAntialias</stylestrategy>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>List OpenFoodFacts</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/assets/images/icon.png</normaloff>:/images/assets/images/icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="local_mode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>false</underline>
             <stylestrategy>NoAntialias</stylestrategy>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>Liste locale</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/assets/images/home 1.png</normaloff>:/images/assets/images/home 1.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="categories_grp">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="title">
        <string>Sélectionnez une catégorie</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListView" name="categories_list">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>10</pointsize>
            <underline>false</underline>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="foods_grp">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="title">
        <string>Sélectionnez un aliment</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListView" name="foods_list">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>10</pointsize>
            <underline>false</underline>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="title">
        <string>Substituts proposés</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTableView" name="substitutes_list"/>
          <widget class="QGroupBox" name="substitute_selected_grp">
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>11</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="title">
            <string>Détails</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QWidget" name="substitute_widget" native="true">
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_nom">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>Nom</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="substitute_name">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_description">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>Description</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_shops">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>Magasins</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QTextEdit" name="substitute_description">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_url">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>url</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="substitute_url">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_score">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>Score</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="substitute_score">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>10</pointsize>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QListView" name="substitute_shops"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="record">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>Enregistrer dans ma base</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>10</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
